### ISSUE TITLE
Option to change _linesLimit on browser side

### ISSUE THREAD

**Issue by Rom1deTroyes** [mthenw/frontail#190]
When I get back to my browser after a night of log scrolling, only the 2000 last line are present, so the logs between 18:30 and 23:00 where lost.

In app.js we got :
```JS
 var _linesLimit = Math.Infinity;
```
redefined in init() by :
```JS
_socket
        .on('options:lines', function(limit) {
          _linesLimit = limit;
        })
```

And the first line is removed if we reach the limit in the `log()` function :
```JS
      if (_logContainer.children.length > _linesLimit) {
        _logContainer.removeChild(_logContainer.children[0]);
      }
```

On my system (Firefox Developper Edition 71.0b5 (64 bits) on Win10), `Math.Infinity` is undefined
```
>> Math.Infinity
<- undefined
```
But _linesLimit is set to 2000 by the socket.io init stuff :
![image](https://user-images.githubusercontent.com/8625432/70616108-065b7b80-1c0e-11ea-9009-195764eaf772.png)
 
As I don't have access to the server side of frontail, but my desktop is capable to display billions of lines, is there a way to change this behaviour without hacking ?

**Comment by mthenw** [comment]
> As I don't have access to the server side of frontail, but my desktop is capable to display billions of lines, is there a way to change this behaviour without hacking ?

I don't think so. Changing this in the frontend is tricky. I will mark it as a feature.
